include/revng/Model/DynamicFunction.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "revng/Model/GeneratedDynamicFunction.h"
+
+/// Function defined in a dynamic library
+class model::DynamicFunction : public model::generated::DynamicFunction {
```

Comment:
FYI: what have does not seem to be a [mixin](https://en.wikipedia.org/wiki/Mixin).